lib/apiservers/engine/backends/portlayer/container_proxy.go

```diff
@@ -248,6 +274,97 @@ func (c *ContainerProxy) CommitContainerHandle(handle, imageID string) error {
 	return nil
 }
 
+// StreamContainerlogs() reads the log stream from the portlayer rest server and writes
+// it directly to the io.Writer that is passed in.
+func (c *ContainerProxy) StreamContainerLogs(name string, out io.Writer, started chan struct{}, showTimestamps bool, followLogs bool, since int64, tailLines int64) error {
+	defer trace.End(trace.Begin("ContainerProxy.StreamContainerLogs"))
```

Comment:
I thought @caglar10ur previously mentioned that trace will automatically provide the function call so this shouldn't include the function but other potentially helpful information.... possibly I misunderstood